collektive-dsl/src/commonMain/kotlin/it/unibo/collektive/aggregate/Field.kt

```diff
@@ -15,13 +15,20 @@ sealed interface Field<ID : Any, out T> {
     /**
      * The [ID] of the local node.
      */
+    @Deprecated("Use local.value instead", replaceWith = ReplaceWith("local.id"))
```

Comment:
```suggestion
    @Deprecated("Use local.id instead", replaceWith = ReplaceWith("local.id"))
```
I think this could be a "typo"